<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnularLT</class>
 <widget class="QDialog" name="AnularLT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnularLT</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>491</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Cedula del LT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leCedula"/>
      </item>
      <item>
       <widget class="QPushButton" name="btBuscarL">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nombre del LT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNombreL">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Anular matr√≠cula a Leader Teacher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAnular">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Anular</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
